packages/space-opera/src/components/materials_panel/materials_panel.ts

```diff
@@ -247,35 +257,76 @@ export class MaterialPanel extends ConnectedLitElement {
     }
   }
 
+  renderVariantsTab() {
+    const hasVariants = getModelViewer().availableVariants.length > 0;
+    return html`
+    <me-expandable-tab tabName="Variants" .open=${hasVariants}>
+      <div slot="content">
+      <me-section-row
+      style='display: ${hasVariants ? '' : 'none'}'>
+        <me-dropdown
+          id="variant-selector"
+          @select=${this.onSelectVariant}
+          >${
+        getModelViewer().availableVariants.map(
+            (name, id) =>
+                html`<paper-item value="${name}">(${id}) ${name}</paper-item>`)}
+        </me-dropdown>
+      </me-section-row label>
+      <div>
+    </me-expandable-tab>
+    `;
+  }
+
   renderSelectMaterialTab() {
     return html`
     <me-expandable-tab tabName="Selected Material" .open=${true} .sticky=${
```

Comment:
If you open enough tabs under this so it scrolls, you'll  see that having a sticky one below a not  sticky one is very weird. Let's go with neither is sticky. Hopefully that will make their  styles match better too.

Reply:
Done.
Looks pretty good IMO.